Changelog.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) (from 3.17.0 onwards)
+
+## [Unreleased]
+
+### Changed
+
+[Deprecation] Adds deprecation warning for all methods containing `master`/`slave` which recommends using the updated `primary`/`replica` methods. The main public methods changed:
+
+1. `on_slave` => `on_replica`
+1. `on_master` => `on_primary`
+
+other methods changed:
+
+1. `on_master_if` => `on_primary_if`
+1. `on_slave_if` => `on_replica_if`
+1. `on_master_unless` => `on_primary_unless`
+1. `on_slave_unless` => `on_replica_unless`
+1. `on_master_or_slave` => `on_primary_or_replica`
```

Comment:
Also, the method `on_master_or_replica` has been removed.

Reply:
the last PR and this will be one release, so `on_master_or_replica` method may never be released?